vagrant-post-script/install_docker.sh

```diff
@@ -15,7 +15,7 @@ sudo apt-key fingerprint 0EBFCD88 || { echo "[Vagrant] ###################     F
 sudo add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/ubuntu bionic stable"
```

Comment:
This should be updated as well, just like all other instances of `bionic`

Reply:
good catch